src/offsetAnchor.js

```diff
@@ -0,0 +1,54 @@
+/* global internalScope */
+
+(function () {
+  var isNumeric = internalScope.isNumeric;
+
+  function offsetAnchorParse (input) {
```

Comment:
Looks like there's a ton of syntax that isn't being supported by this function.
Please add a TODO to make it clear this is the case.